test/webRDFS/WebServerTest.cc

```diff
@@ -45,6 +45,41 @@ TEST(WebServerTest, testRead) {
   system("hdfs dfs -fs hdfs://localhost:5351 -rm /fileToRead");
 }
 
+TEST(WebServerTest, testCreate) {
+  system(
+        "hdfs dfs -fs hdfs://localhost:5351 "
+        "-copyFromLocal fileForTesting /fileToRead");
+
+      ASSERT_EQ(0,
+                system("curl -i --insecure https://localhost:8080/webhdfs/v1/"
+                               "fileToRead?op=CREATE > actualResultRead"));
```

Comment:
Also change the name of the actualResultRead / expectedResultRead to actualResultCreate and expectedResultCreate because as of now the test will fail because the result of a read isn't the same as create